CHANGES.md

```diff
@@ -114,6 +114,10 @@ resuming a workflow.
 [#5625](https://github.com/cylc/cylc-flow/pull/5625) - Exclude `setuptools`
 version (v67) which results in dependency check failure with editable installs.
 
+[#5652](https://github.com/cylc/cylc-flow/pull/5562) - Prevent the same
+message being logged multiple times when polled.
+
+
```

Comment:
Needs to be moved now. I would suggest writing the description to `changes.d/5562.fix.md` in anticipation of #5634

Reply:
It's waiting on Hilary, who will presumably want to have a think about town crier (unless he doesn't). I'll update in this format for now.

Reply:
IMO it's easier to do towncrier fragments now and switch back to CHANGES.md later in the chance we don't proceed with towncrier.

Reply:
#5634 has been merged anyway so please do convert to a towncrier fragment